### DISCUSSION TITLE
Easy to miss that approving a PR will auto-merge

### DISCUSSION THREAD

**Discussion by iainlane** [community/community#54248] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When submitting an approval it would be nice to know if the PR is going to be automerged. Often I will leave nits or other feedback and I would like the author to consider them before merging. If I notice that automerge is on then I will turn it off but I find that fact is presented too subtly at the minute so I often miss it.

Suggest a change to the review submission itself to make it clear that your approval will also merge the branch, either the radios or the "Submit review" button itself.

**Comment by taozhou-glean** [comment]
or provide some option to only allow enable `auto-merge` option after all or some requirements satisfied at least once

most of times, I want to enable auto-merge only after approvals received from my teammates, so I can just merge it without having to check back once CI finished
